server/controllers/appointment.js

```diff
@@ -0,0 +1,39 @@
+const appointmentsRouter = require("express").Router();
+const { Appointment } = require("../models/");
+
+// Create Appointments
+// How to make this an authenticated route?
+appointmentsRouter.post("/", (request, response, next) => {
+    const body = request.body;
+    const time = new Date(body.time);
+    console.log(typeof time)
+    console.log(time);
+    const appointment = new Appointment({
+        name:       body.name,
+        email:      body.email,
+        time:       time,
+        timezone:   body.timezone
+    });
+
+    // Need to add logic for connecting Appointments to users meeting
+
+    appointment.save()
+        .then(savedAppointments => {
+            console.log(savedAppointments);
+            response.json(savedAppointments);
+        })
+        .catch(error => {
+            console.error(error);
+            response.json(error);
+            next(error)
+        });
+});
+
+// Fetch/Read All Appointmentss
+appointmentsRouter.get("/", (request, response) => {
+    Appointment.find({}).then(Appointments => {
```

Comment:
```suggestion
    Appointment.find({}).then(appointments => {
```

Reply:
I would reserve capitalized variables for component and model names, since this variable is rather a callback return value, i recommend just naming it `appointments`

Reply:
looks like you did this for other controllers :) so it's just this one

Reply:
yeah, I think that was by mistake when I did a (Ctrl + D) to edit multiple "appointment" text and didn't make some of them lowercase.